setup.cfg

```diff
@@ -10,3 +10,13 @@ testpaths = subhub
 
 [pytest]
 addopts = -p no:warnings
+
+[mypy]
+disallow_untyped_calls = True
+follow_imports = normal
+ignore_missing_imports = True
+python_version = 3.7
+strict_optional = True
+warn_no_return = True
+warn_redundant_casts = True
+warn_unused_ignores = True
```

Comment:
`warn_unreachable`?

Reply:
Added `warn_unreachable = True`  `allow_redefinition` is defaulted to false, which looks to be what would be wanted.  "Allows variables to be redefined with an arbitrary type, as long as the redefinition is in the same block and nesting level as the original definition."